__mocks__/mockarticles.js

```diff
@@ -0,0 +1,95 @@
+        source: "The Times",
+        image: "https://www.thetimes.co.uk/d/assets/society-69df0d1a8d.png",
+        region: "Europe"
+        },
+        'PN7QJZEP4K': {
+        title: "Payback time: eco upgrades based on the new energy price cap",
+        url: "https://12ft.io/https://www.thetimes.co.ukhttps://www.thetimes.co.uk/article/how-to-save-energy-at-home-reduce-energy-bills-8bv763fn6",
+        source: "The Times",
+        image: "https://www.thetimes.co.uk/d/assets/society-69df0d1a8d.png",
+        region: "Europe"
+        },
+        'ZM5IADXOO5': {
+        title: "High winds and heavy rainfall generate record for renewable energy",
+        url: "https://12ft.io/https://www.thetimes.co.ukhttps://www.thetimes.co.uk/article/high-winds-and-heavy-rainfall-generate-record-for-renewable-energy-9xrvd669v",
+        source: "The Times",
+        image: "https://www.thetimes.co.uk/d/assets/society-69df0d1a8d.png",
+        region: "Europe"
+        },
+        'T6KGISBFM5': {
+        title: "See our renewable energy page",
+        url: "https://12ft.io/https://www.thetimes.co.ukhttps://www.thetimes.co.uk/environment/renewable-energy",
+        source: "The Times",
+        image: "https://www.thetimes.co.uk/d/assets/society-69df0d1a8d.png",
+        region: "Europe"
+        },
+        '3E2NLIZK26': {
+        title: "Green power firms face windfall tax to fund energy bills",
+        url: "https://12ft.io/https://www.thetimes.co.ukhttps://www.thetimes.co.uk/article/green-power-firms-face-windfall-tax-to-fund-lower-energy-bills-8lf5c5sz8",
+        source: "The Times",
+        image: "https://www.thetimes.co.uk/d/assets/society-69df0d1a8d.png",
+        region: "Europe"
+        },
+        '7TMMCXPUFQ': {
+        title: "Mild winter forecast offers hope on energy bills",
+        url: "https://12ft.io/https://www.thetimes.co.ukhttps://www.thetimes.co.uk/article/mild-winter-forecast-offers-hope-on-energy-bills-l0lxcqngs",
+        source: "The Times",
+        image: "https://www.thetimes.co.uk/d/assets/society-69df0d1a8d.png",
+        region: "Europe"
+        },
+        'LUYXFMPRJH': {
+        title: "No 10 rejects ‘bespoke’ energy-saving push but will boost existing messages",
+        url: "https://12ft.io/https://www.theguardian.com/business/2022/oct/10/no-10-rejects-bespoke-energy-saving-push-but-will-boost-existing-messages",
+        source: "The Guardian",
+        image: "https://assets.guim.co.uk/images/eada8aa27c12fe2d5afa3a89d3fbae0d/fallback-logo.png",
+        region: "Europe"
+        },
+        'PN7QJPEP4K': {
+            title: "Payback time: eco upgrades based on the new energy price cap",
+            url: "https://12ft.io/https://www.thetimes.co.ukhttps://www.thetimes.co.uk/article/how-to-save-energy-at-home-reduce-energy-bills-8bv763fn6",
+            source: "The Times",
+            image: "https://www.thetimes.co.uk/d/assets/society-69df0d1a8d.png",
+            region: "Europe"
+            },
```

Comment:
The formatting is a little weird in this file **BUT** it is still valid JSON so I shouldn't complain. :sweat_smile: